index.js

```diff
@@ -516,7 +518,7 @@ class HtmlWebpackInlineSVGPlugin {
             return name !== 'inline'
                 && name !== 'src'
                 ? acc + `${name}="${value}" `
-                : ''
+                : acc
```

Comment:
This is a bug. The accumulator must always be returned. If not what was generated on the previous iteration will be forgotten. I had to update tests because of this.